.github/workflows/publish.yaml

```diff
@@ -0,0 +1,91 @@
+# Run locally with act:
+#
+# act workflow_dispatch \
+#  --input command=<test|publish> \
+#  --input version=<version> \
+#  --workflows ./.github/workflows/release.yaml \
+#  --env-file <(aws configure export-credentials --profile [aws-profile] --format env)
+
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      command:
+        required: true
+        type: choice
+        values:
+          - test
+          - publish
+        default: test
+      version:
+        required: true
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: configure git
+        shell: bash
+        run: |
+          git config --global user.name "FusionAuth Automation"
+          git config --global user.email "donotreply@fusionauth.io"
+
+      - name: set aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.ACTIONS_AWS_ROLE_ARN }}
+          role-session-name: aws-auth-action
+          aws-region: us-west-2
+
+      - name: get secret
+        run: |
+          while IFS=$'\t' read -r key value; do
+            echo "::add-mask::${value}"
+            echo "${key}=${value}" >> $GITHUB_ENV
+          done < <(aws secretsmanager get-secret-value \
+            --region us-west-2 \
+            --secret-id platform/npmjs \
```

Comment:
This looks like we have one secret for NPM? Does it make sense to have multiple secrets for each project?

Reply:
I would prefer not to, but I'm open to hearing arguments in favor